addons/website_forum/tests/test_forum_internals.py

```diff
@@ -58,6 +58,23 @@ def test_website_forum_count(self):
         self.assertEqual(website_2.forum_count, 6,
                          '6 global forums')
 
+    def test_website_forum_last_post_id(self):
+        """Check that each forum's last post is computed correctly."""
+        test_forums = self.base_forum | self.forum
+        new_posts = self.env["forum.post"].create([{
+            'name': f'New Post {forum_post_idx}',
+            'forum_id': forum.id,
+        } for forum_post_idx, forum in enumerate(test_forums)])
+
+        self.assertEqual(test_forums.last_post_id.ids, new_posts.ids)
```

Comment:
Additionally you could assertQueryCount(1) on this specific line, as this change is about performances more than the value itself?